network.env.dist

```diff
@@ -1,14 +1,18 @@
 #Unique Coinbase addresses
 
-ZONE_0_0_COINBASE=0x04a3e45aa16163F2663015b6695894D918866d19
-ZONE_0_1_COINBASE=0x21c7650E65b164B2ab645eAF1141b569B2c82Bd7
-ZONE_0_2_COINBASE=0x3e742F0AE63d62304153526A51EE8BF0531d1887
-ZONE_1_0_COINBASE=0x72c871f639ed156De0b97C1b533e2617730b7ec2
-ZONE_1_1_COINBASE=0x755c3603c5688CF3105F1a1AfC11fa0e1981b03B
-ZONE_1_2_COINBASE=0x9fF7B1A33BB22b70F5fb78A420CEd9075C437c87
-ZONE_2_0_COINBASE=0xb8094B2bc411942fd078D994Cc4e9418D6A5B071
-ZONE_2_1_COINBASE=0xC83cb918dd9267a344B51f57e28e8cf977057E05
-ZONE_2_2_COINBASE=0xF39E7d05B5A1a2F934cC43221383f29e4794c822
+PRIME_COINBASE=0x00114a47a5d39ea2022dd4d864cb62cfd16879fc
```

Comment:
I don't think you meant to tweak coinbase addresses in this commit. The prior commit should be dropped (not relevant to this PR) and these changes will no longer be necessary.